tools/fragpipe/macros.xml

```diff
@@ -0,0 +1,1206 @@
+                            <option value="true">Yes</option>
+                            <option value="false">No</option>
+                        </param>
+                        <param name="print_RefInt" type="select" optional="true" label="print individual reference sample abundance estimates for each multiplex in the final reports">
+                            <option value="false">No</option>
+                            <option value="true">Yes</option>
+                        </param>
+                    </section>
+                </when>
+            </conditional>
+        </section>
+    </xml>
+
+    <!-- Set isobaric quantification workflow options -->
+    <token name="@ISOBARIC_QUANTIFICATION@"><![CDATA[
+        #set $prfx = $wf.quant_iso.isobaric_quantification
+        #if $prfx.isobaric_quantification_step == 'no'
+            #set $wfdict['tmtintegrator.run-tmtintegrator'] = 'false'
+        #elif $prfx.isobaric_quantification_step == 'yes'
+            #set $wfdict['tmtintegrator.run-tmtintegrator'] = 'true'
+            #set $wfdict['tmtintegrator.dont-run-fq-lq'] = 'false'
+            #set $cxt = $prfx.options
+            #set $wfdict['tmtintegrator.quant_level'] = $cxt.quant_level
+            #set $wfdict['tmtintegrator.channel_num'] = $cxt.channel_num
+            #if $cxt.ref_tag == 'Virtual'
+                #set $wfdict['tmtintegrator.add_Ref'] = -1
+            #else
+                #set $wfdict['tmtintegrator.add_Ref'] = 1
+                #set $wfdict['tmtintegrator.ref_tag'] = $cxt.ref_tag
+            #end if
+            #if $cxt.groupby != ''
+                #set $wfdict['tmtintegrator.groupby'] = $cxt.groupby
+            #end if
+            #set $wfdict['tmtintegrator.log2transformed'] = $cxt.log2transformed
+            #if $cxt.prot_norm != 'None'
+                #set $wfdict['tmtintegrator.prot_norm'] = $cxt.prot_norm
+            #end if
+            #if cxt.glycan_conditional.use_glycan_composition == 'yes'
+                #if $cxt.glycan_conditional.mod_tag != 'None'
+                    #set $wfdict['tmtintegrator.mod_tag'] = $cxt.glycan_conditional.mod_tag
+                #end if
+                #if $cxt.glycan_conditional.min_site_prob != ''
+                    #set $wfdict['tmtintegrator.min_site_prob'] = $cxt.glycan_conditional.min_site_prob
+                #end if
+                #if $cxt.glycan_conditional.glyco_qval != ''
+                    #set $wfdict['tmtintegrator.glyco_qval'] = $cxt.glycan_conditional.glyco_qval
+                #end if
+            #end if
+            #if $cxt.unique_gene != 'None'
+                #set $wfdict['tmtintegrator.unique_gene'] = $cxt.unique_gene
+            #end if
+            #if $cxt.unique_pep != 'None'
+                #set $wfdict['tmtintegrator.unique_pep'] = $cxt.unique_pep
+            #end if
+            #if $cxt.min_pep_prob != ''
```

Comment:
Not 100% sure, but I think you should test for `str($cxt.min_pep_prob) != ''` . Needs to be checked everywhere.